README.md

```diff
@@ -24,8 +24,27 @@ high-performance, block-structured adaptive mesh refinement. Emu is
 parallelized with MPI + OpenMP for CPUs and MPI + CUDA for GPUs.
 
 # Getting Started From Scratch
+If you would like to run Emu on your own machine, there are some modules
+you will need to install depending on your operating system.
 
-If you would like to run Emu on your own machine, first clone Emu with the AMReX submodule:
+## Linux or WSL
+
+If you are running Emu on Linux or a WSL, the commands you need to
+write in the terminal are:
+
+```
+sudo apt-get install g++ libopenmpi-dev python-3 gfortran gnuplot-x11
```

Comment:
I would remove 'sudo' - this gives users special permission, and anybody who is using sudo will know they need to do so.